app/config/Seeds/UsersSeed.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Seed\AbstractSeed;
+
+/**
+ * Users seed.
+ */
+class UsersSeed extends AbstractSeed
+{
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeds is available here:
+     * https://book.cakephp.org/phinx/0/en/seeding.html
+     *
+     * @return void
+     */
+    public function run(): void
+    {
+        try {
+            // Insert data into the 'users' table
+            $usersData = [
+                [
+                    'email' => 'cakephp@example.com',
+                    'password' => 'secret',
+                    'created' => date('Y-m-d H:i:s'),
+                    'modified' => date('Y-m-d H:i:s'),
+                ],
+            ];
+            $this->table('users')->insert($usersData)->save();
+        } catch (\Exception $e) {
+            echo "UsersSeed already executed";
+            echo "Error: " . $e->getMessage();
+        }
+
```

Comment:
Similar comment about try & catch and logging.

Reply:
changed as well